app/src/ui/business-card/BusinessCard.module.scss

```
@@ -0,0 +1,209 @@
+@import "src/theme/base";
+
+$card-border: #e6ebf0;
+$card-background: #ffff;
+$card-subtitle: #898f99;
+
+$card-width: 253px;
+$card-height: 295px;
+$card-default-bottom: 0.2rem;
+
+@mixin withContent {
+  position: relative;
+  top: 120px;
+  height: 500px;
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+  padding-right: 1rem;
+  padding-left: 1rem;
+  background: $card-background;
+}
+
+@mixin withInvestmentInfo {
+  position: absolute;
+  left: 0;
+  right: 0;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+@mixin withInvestmentValue {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex: 0 0 50%;
+  max-width: 50%;
+  width: 100%;
+  flex-wrap: wrap;
+  align-items: center;
+
+  h5 {
+    font-size: $font-size-5;
+    font-weight: $font-weight-bold;
+    line-height: 1rem;
+    margin-top: 0;
+    margin-bottom: $card-default-bottom;
+  }
```

```suggestion
  h5 {
    @include font-properties($font-size-headline-5);
    margin-top: 0;
    margin-bottom: $card-default-bottom;
  }
```

Vars are stored in `_variables.scss`